src/parser/mod.rs

```diff
@@ -3605,6 +3605,24 @@ impl<'a> Parser<'a> {
         Ok(values)
     }
 
+    pub fn parse_period_separated<T, F>(&mut self, mut f: F) -> Result<Vec<T>, ParserError>
```

Comment:
```suggestion
    fn parse_period_separated<T, F>(&mut self, mut f: F) -> Result<Vec<T>, ParserError>
```
Can we add a short description mentioning what the function does?